source/introduction/index.rst

```diff
@@ -0,0 +1,43 @@
+Introduction
+============
+
+
+In this `IEEE EMBC 2023 <https://embc.embs.org/2023/>`_ workshop, we will demonstrate a range of tools that encourage users to create, share, and publish physiological models in a reproducible and FAIR manner.
+We will show how using appropriate standard formats and consistently adopting suitable practices eases the transition from reproducible model to reproducible publication, and begins to provide evidence of model credibility that is crucial for the translation of models into clinical applications.
+We will demonstrate examples which are beginning to achieve such translation of models into clinical workflows designed to support specific clinical use-cases.
+
+The workshop will use free and open-source software tools being developed for the `12 Labours project <https://tinyurl.com/12labours>`_, `SPARC <https://sparc.science>`_, and the `Center for Reproducible Biomedical Modeling <https://reproduciblebiomodels.org>`_.
+We will provide hands-on tutorials using stand-alone desktop or web applications and using services or programming interfaces that could be more easily integrated into existing tooling and practices in your own lab.
+Workshop facilitators will be on hand to help with installation and usage questions, and in-depth discussion regarding potential integration of the tools.
+All tools and example models and data used in the workshop will be available via `this website <https://tools-for-credible-digital-twins.github.io/>`_ for attendees to access and share, along with suitable documentation to help attendees share what they have learnt with colleagues.
+
+To help encourage the publication of reproducible and credible models, we have launched the `Physiome <https://journal.physiomeproject.org>`_ journal.
+In this workshop, we will discuss recent developments with *Physiome* as we look to make the articles dynamic and interactive '*active papers*' and show how the tools and practices covered throughout the workshop can lead to the publication of your work in this way with minimal additional effort.
```

Comment:
I wouldn't mind avoiding the use of `additional` here.  Can we say using the tools and practices combined with the underlying standards can lead to the publication of your work in this way? These *active papers* will be the building blocks of credible physiological modelling and translational science?

Reply:
see 7ba4224 to see if that is along the lines of what you were thinking.